Baconit/Strings/es-es/Resources.resw

```diff
@@ -0,0 +1,889 @@
+  <data name="TimeTB.Text" xml:space="preserve">
+    <value>Hora:</value>
+  </data>
+  <data name="ViewContextTB.Text" xml:space="preserve">
+    <value>Ver en contexto</value>
+  </data>
+  <data name="CLinkFMI.Text" xml:space="preserve">
+    <value>Copiar enlace</value>
+  </data>
+  <data name="CMBest.Text" xml:space="preserve">
+    <value>Lo mejor</value>
+  </data>
+  <data name="CMContro.Text" xml:space="preserve">
+    <value>Controvérsico</value>
+  </data>
+  <data name="CMNew.Text" xml:space="preserve">
+    <value>Nuevo</value>
+  </data>
+  <data name="CMOld.Text" xml:space="preserve">
+    <value>Viejo</value>
+  </data>
+  <data name="CMQA.Text" xml:space="preserve">
+    <value>Q&amp;A</value>
+  </data>
+  <data name="CMTop.Text" xml:space="preserve">
+    <value>Lo mejor</value>
+  </data>
+  <data name="collapseTBN.Text" xml:space="preserve">
+    <value>Unir</value>
+  </data>
+  <data name="CopyPerFMI.Text" xml:space="preserve">
+    <value>Copiar el enlace</value>
+  </data>
+  <data name="CountTB.Text" xml:space="preserve">
+    <value>Contar:</value>
+  </data>
+  <data name="CPermaFMI.Text" xml:space="preserve">
+    <value>Copiar el enlace</value>
+  </data>
+  <data name="DeletePostClick.Text" xml:space="preserve">
+    <value>Eliminar el post</value>
+  </data>
+  <data name="EditPostClick.Text" xml:space="preserve">
+    <value>Editar el post</value>
+  </data>
+  <data name="MoreTB.Text" xml:space="preserve">
+    <value>Más</value>
+  </data>
+  <data name="RunGilded.Text" xml:space="preserve">
+    <value>Exclusiva</value>
+  </data>
+  <data name="RunSticky.Text" xml:space="preserve">
+    <value>Pegado</value>
+  </data>
+  <data name="ShareCommentFMI.Text" xml:space="preserve">
+    <value>Compartir comentario</value>
+  </data>
+  <data name="SharePostFMI.Text" xml:space="preserve">
+    <value>Compartir post</value>
+  </data>
+  <data name="TapViewEntire.Text" xml:space="preserve">
+    <value>Pulsa en aquí para ver el hilo completo</value>
+  </data>
+  <data name="AboutDescription.Text" xml:space="preserve">
+    <value>Reddunt es un maravilloso cliente de Reddit basado en Baconit, una aplicación de código libre. Está disponible para todos los dispositivos con Windows 10. 
+El principal objetivo es la optimización y la utilización de características propias de Windows 10!</value>
+  </data>
+  <data name="AddMessagesTB.Text" xml:space="preserve">
+    <value>Añadir mensajes al centro de notificaciones silenciosamente.</value>
+  </data>
+  <data name="AlwaysCombo.Content" xml:space="preserve">
+    <value>Siempre</value>
+  </data>
+  <data name="AuthorFilter.PlaceholderText" xml:space="preserve">
+    <value>Filtro por autor.</value>
+  </data>
+  <data name="AutoTB.Text" xml:space="preserve">
+    <value>Minimizar la cabecera de ede los comentarios.</value>
+  </data>
+  <data name="AutoUpdateTB.Text" xml:space="preserve">
+    <value>Automáticamente actualiza la pantalla de bloqueo.</value>
```

Comment:
I would change the order to "Actualiza automáticamente..."

Reply:
My mistake